packages/grafana-ui/src/components/Forms/InlineLabel.mdx

```diff
@@ -0,0 +1,15 @@
+import { Props } from "@storybook/addon-docs/blocks";
+import { InlineLabel } from "./InlineLabel";
+
+# InlineLabel
+
+A horizontal variant of Label, primarily used in query editors.
```

Comment:
I feel like the limitations of the InlineLabel could be explained a bit more here: 
- There is no description option, if you need to add additional explanation use the tooltip variant. 
- For query editor readability, the label text should be as short as possible (4 words or less).
- Which components can the label be combined with and how does that work (`InlineField`)?